store/src/test/java/org/apache/rocketmq/store/timer/TimerMessageStoreTest.java

```diff
@@ -211,6 +224,16 @@ public Boolean call() {
         }
     }
 
+    @Test
+    public void testPutTimerMessage1() throws Exception {
+        testPutTimerMessageCommon();
+    }
+
+    @Test
+    public void testPutTimerMessage2() throws Exception {
+        testPutTimerMessageCommon();
+    }
```

Comment:
It is also possible that you will not run DLedger mode after running twice. It is recommended to set parameters or flag bits to determine which mode to run.

Reply:
done